packages/celest/example/README.md

```diff
@@ -0,0 +1,8 @@
+# hello
+
+A new Flutter project.
+
+## Running the app
+
+- Go to the hello directory, and run `flutter pub get`
+- Go to the `celest` directory inside the hello example app and run `dart pub get`
```

Comment:
just wondering here on why you need to run dart pub get if the CLI takes care of installing dependencies for you?